dear_petition/petition/export/forms.py

```diff
@@ -138,6 +139,10 @@ def map_offenses(self):
             self.data[f"DOOF:{i}"] = self.format_date(ciprs_record.offense_date)
             self.data[f"DismissalDate:{i}"] = self.format_date(offense.disposed_on)
 
+    def map_additional_forms(self):
+        if self.petition.has_attachments():
+            self.data[f"CkBox_Attchmt"] = Checkbox("Yes")
```

Comment:
Is the format string necessary for `CkBox_Attchmt`?

Reply:
No, I'll remove that.